composer.json

```diff
@@ -1,4 +1,6 @@
 {
+  "name": "mediatis/formrelay-salesforce-case",
+  "description": "Add description...",
```

Comment:
@mkarulin Stealing from mediatis/formrelay-salesforce I would recommend this description:

Delivery system for Salesforce (Web-To-Case-API) based on Mediatis Formrelay